Makefile

```diff
@@ -82,17 +82,23 @@ ifeq ($(MAKECMDGOALS),rocksdbjavastatic)
 endif
 
 ifeq ($(MAKECMDGOALS),rocksdbjavastaticrelease)
-	DEBUG_LEVEL=0
+	ifneq ($(DEBUG_LEVEL),2)
```

Comment:
I believe nobody explicitly would to set DEBUG_LEVEL=1 if they are really debugging, they would set DEBUG_LEVEL=2. So this should be ok.